demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>...to view how the loss function decreases and accuracy increases (hopefully!) over the training epochs:</blockquote><p> </p><p>Maybe be a bit more neutral here (no need to be hopeful of an outcome) and say something like, "we can examine the behavior of the loss function and training metrics by visualizing their values during training using <code>StellarGraph</code>'s  <code>plot_history</code>  utility funtion."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>